assets/js/common/OperationModals/SaptuneSolutionApplyModal.test.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { act, render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import { noop } from 'lodash';
```

Comment:
I guess adding noop to the components, will not require it in the tests anymore?